Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -363,9 +379,28 @@ public class HTTPClient {
 
         /// Event Loop will be selected by the library.
         public static let indifferent = EventLoopPreference(.indifferent)
+
         /// Library will try to use provided event loop if possible.
+        @available(*, deprecated, renamed: "delegate(on:)")
```

Comment:
do we really need this?

Reply:
@artemredkin so far, we've broken everybody in every new alpha release which is a real issue. I think we should do this and remove before 1.0.0. I'll file an issue to remove all deprecated stuff just before then 1.0.0 release

Reply:
@artemredkin happy with that?